testnet/setup.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
```

Comment:
Can you rename directory to eth2_testnet

Reply:
or maybe better lighthouse_testnet. As we will add up other clients later as well

Reply:
> e --dummy-eth1 I think that flag is missing on beacon nodes. But I think we can fully reuse those scripts

and I use this flag ins `start-beacon-node.sh`
```
#! /bin/bash

exec lighthouse --debug-level $DEBUG_LEVEL beacon_node \
  --dummy-eth1 --disable-discovery \
  --http --http-address 0.0.0.0 \
  --testnet-dir /root/testnet \
  --datadir /root/beacon \
  --enr-address 127.0.0.1 --enr-udp-port 9000 --enr-tcp-port 9000
```

Reply:
then better will be to nest inside `testnet` => `lighthouse` and inside every folder own `docker-compose.yml` or somelike this

Reply:
yeah that works too